api/src/main/java/org/openmrs/OrderSet.java

```diff
@@ -40,6 +40,8 @@
 	
 	private List<OrderSetMember> orderSetMembers;
 	
+	private Category category;
```

Comment:
The category field should of of type or class Concept

Reply:
Let me have that changed.